packages/tasks/src/model-libraries.ts

```diff
@@ -469,6 +469,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"nemo" OR path:"model_config.yaml"`,
 	},
+	open_oasis: {
+		prettyLabel: "open-oasis",
+		repoName: "open-oasis",
+		repoUrl: "https://github.com/etched-ai/open-oasis",
+		filter: true,
```

Comment:
```suggestion
```

This would make `open-oasis` a listed library on https://huggingface.co/models which we want to avoid for small or niche libraries (requires a least a few thousands models to get listed)